### PR TITLE
fix some typos

### PR DESCRIPTION
**Pull request by MizukiCry**
### What problem does this PR solve?

Issue Number: #xxx 

Problem Summary: fix some typos.

### What is changed and how it works?

What's Changed:

How it Works:

Side effects(Breaking backward compatibility? Performance regression?):

### Check List

- [x] Relevant documentation/comments is changed or added
- [x] I acknowledge that all my contributions will be made under the project's license

### CODE REVIEW

File: docs/en/snapshotcloneserver_en.md [hunk 1]
```diff
@@ -76,7 +76,7 @@ Please refer to the [document](../cn/snapshotcloneserver_interface.md) of snapsh
 
 ### 2.1 Intro to Curve Clone Module
 
-In section 1 we introduced the snapshot system, which is for creating snapshots of  files of curvefs and incrementally dump the data to S3. But for a complete snapshot system, supporting only the snapshot creation is not enough, file recovery and cloning is the reason why take snapshot. In this section we introduce Curve cloning systerm(recovering can be considered as cloning to some extent).
+In section 1 we introduced the snapshot system, which is for creating snapshots of  files of curvefs and incrementally dump the data to S3. But for a complete snapshot system, supporting only the snapshot creation is not enough, file recovery and cloning is the reason why take snapshot. In this section we introduce Curve cloning system(recovering can be considered as cloning to some extent).
 
 According to the data source, there are two kinds of cloning, including cloning from snapshot and cloning from image, and if dividing by whether the data is entirely cloned before the service is provided, it can be divided into Lazy Clone and Not-Lazy Clone.
```
[the remark this answers is not in the document]
Comment (aspirer) [reply docs/en/snapshotcloneserver_en.md:82]: @MizukiCry the bot find a new typo, can you fix it in this pr? the first one above.

[the remark this answers is not in the document]
Comment (MizukiCry) [reply docs/en/snapshotcloneserver_en.md:82]: > @MizukiCry the bot find a new typo, can you fix it in this pr? the first one above.

@aspirer It's already fixed.

**Comment by ilixiaocui** [comment]
@Cyber-SiKu 
Please help reveiw this pr.

**Review by aspirer — APPROVED** [review]

**Review by Cyber-SiKu — APPROVED** [review]

**Comment by Cyber-SiKu** [comment]
cicheck

**Comment by opencurveadmin** [comment]
@MizukiCry Welcome to join the contributor family, add community contact information and there will be a gift bag. 

image

**Pull request merged**